tests/small_numbers/float32_lib.ml

```diff
@@ -230,13 +235,24 @@ let () =
     bit_eq (F32.copy_sign f1 f2) (CF32.copy_sign f1 f2);
     bit_eq (F32.min f1 f2) (CF32.min f1 f2);
     bit_eq (F32.max f1 f2) (CF32.max f1 f2);
+    bit_eq (F32.With_weird_nan_behavior.min f1 f2) (CF32.min_weird f1 f2);
+    bit_eq (F32.With_weird_nan_behavior.max f1 f2) (CF32.max_weird f1 f2);
     bit_eq (F32.min_num f1 f2) (CF32.min_num f1 f2);
     bit_eq (F32.max_num f1 f2) (CF32.max_num f1 f2);
     assert((F32.compare f1 f2) = (CF32.compare f1 f2));
     assert((F32.equal f1 f2) = (CF32.equal f1 f2));
   )
 ;;
 
+let () =
+  CF32.check_float32s (fun f _ ->
+    bit_eq (F32.round_up f) (CF32.ceil f);
+    bit_eq (F32.round_down f) (CF32.floor f);
+    bit_eq (F32.round_half_to_even f) (CF32.round_current f);
+    assert ((F32.iround_half_to_even f) = (CF32.iround_current f));
```

Comment:
I think there should be a comment
explaining why this line is different.

Reply:
Added